tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -885,6 +885,9 @@ def cleanup_gaia(self, full_reset=True):
         # disable sound completely
         self.data_layer.set_volume(0)
 
+        # disable search suggestions
+        self.data_layer.set_setting('search.suggestions.enabled', False)
```

Comment:
+1 I think this might be one of the issues we saw some intermitent fails in the past